src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,38 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__event-multiday {
+		background-color: rgba(var(--color-accent-primary), .1);
+		border-radius: 6px;
+		border-style: solid;
+		border-width: 0 2px 0px 2px;
```

Comment:
We're going to need to allow for various cases, such as when a multi-day event spans multiple weeks. In this case, the end of the first week will not have a border on the end, the middle weeks will not have a border on either end, and the last week will not have a border at the start. I would suggest having class modifiers to apply each border separately.